target/linux/ramips/image/mt76x8.mk

```diff
@@ -368,7 +368,7 @@ TARGET_DEVICES += keenetic_kn-1613
 define Device/kroks_kndrt31r16
   IMAGE_SIZE := 16064k
   DEVICE_VENDOR := Kroks
-  DEVICE_MODEL := Rt-Cse5 UW DRSIM
+  DEVICE_MODEL := Rt-Cse SIM Injector DS
   DEVICE_ALT0_VENDOR := Kroks
   DEVICE_ALT0_MODEL := KNdRt31R16
```

Comment:
By the way. The model name in dts is `model = "Kroks KNdRt31R16";`.

Reply:
That's true, but here we use more user-friendly name that can be found on Wiki/official site.